src/components/FaucetDropdown/FaucetDropdown.css

```diff
@@ -0,0 +1,63 @@
+.menu-itemss:focus-visible {
```

Comment:
Perhaps this should be `menu-items` instead in all its references:

`menu-itemss => menu-items`

Reply:
changes it to menu
menu-itemss => menu